workflow-examples/run_examples.sh

```diff
@@ -263,4 +310,5 @@ run_complex_flow() {
   echo "network workflow execution id:" $(echo_green $EXECUTION_ID)
 }
 
-run_complex_flow
+run_simple_flow
+# run_complex_flow
```

Comment:
perhaps make this configurable with 'simple' as default?
e.g. For running the simple flow:
```
workflow-examples/run_examples.sh simple
# or
workflow-examples/run_examples.sh
```
and for complex:
```
workflow-examples/run_examples.sh complex
```
that way there won't be a need to modify the file when using it

Reply:
Added argument to control the flow type
Default type is complex, to keep previous behavior